std/math/hardware.d

```diff
@@ -0,0 +1,23 @@
+// Written in the D programming language.
+
+/**
+This package is currently in a nascent state and may be subject to
+change. Please do not use it yet, but stick to $(MREF std, math).
+
+Copyright: Copyright The D Language Foundation 2000 - 2011.
+           D implementations of tan, atan, atan2, exp, expm1, exp2, log, log10, log1p,
+           log2, floor, ceil and lrint functions are based on the CEPHES math library,
+           which is Copyright (C) 2001 Stephen L. Moshier $(LT)steve@moshier.net$(GT)
```

Comment:
We don't need a reference to the Cephes library in modules where these functions don't even exist.

i.e: Mention tan, atan, atan2 only in `std.math.trigonometry`; floor, ceil, lrint only in `std.math.rounding`, etc.

Reply:
When you told me yesterday at beerconf, I was too tired to change this immediately. Now a new version is online.

Reply:
That's fine, I thought I'd just write a comment for posterity though.